shinydashboard/lantern/server.R

```diff
@@ -81,15 +87,15 @@ function(input, output, session) {
   )
 
   show_filter <- reactive(
-    input$side_menu %in% c("endpoints_tab", "capability_tab", "fields_tab", "security_tab", "smartresponse_tab", "location_tab")
+    input$side_menu %in% c("endpoints_tab", "capability_tab", "fields_tab", "security_tab", "smartresponse_tab", "location_tab", "values_tab")
   )
 
-  show_http_vendor_filter <- reactive(input$side_menu %in% c("dashboard_tab"))
```

Comment:
Was this not doing anything before?

Reply:
i think there were just some mix-ups at some point with a rebase because line 74 is identical, same with the `show_datefilter` that seems to be a misspelling of `show_date_filter` that exists on line 93